packages/paste-website/src/pages/layout/media-object/index.mdx

```diff
@@ -0,0 +1,287 @@
+  {`<MediaObject as="div">
+  <MediaFigure
+    as="div"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+</MediaObject>`}
+</LivePreview>
+
+### Vertical alignment
+
+By default the `MediaBody` is top aligned to the `MediaFigure`. By settting the `verticalAlign` prop on the `MediaObject` to "center" you can align the `MediaBody` to the middle of the `MediaFigure`.
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaFigure
+    as="div"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+</MediaObject>`}
+</LivePreview>
+
+### MediaFigure at the end
+
+To place the `MediaFigure` after the `MediaBody`, at the end of the `MediaObject`, place the `MediaFigure` after the `MediaBody` in the DOM. Be sure to set the `align` prop on the `MediaFigure` to "end" to reverse the spacing.
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+  <MediaFigure
+    as="div"
+    align="end"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+</MediaObject>`}
+</LivePreview>
+
+### Double figure
+
+To have a `MediaFigure` at either end of the `MediaObject` add a `MediaFigure` before and after the `MediaBody`, setting the second `MediaFigure` to `align` "end".
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaFigure
+    as="div"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+  <MediaFigure
+    as="div"
+    align="end"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+</MediaObject>`}
+</LivePreview>
+
+## Paste Examples
+
+Places where you are likely to see this pattern used in Paste include:
+
+### The Alert component
+
+The Alert is a classic Media Object layout. The Status level icon is the Media Figure, and the alert description is the Media Body.
+
+<LivePreview scope={{Alert}} language="jsx">
+  {`<Alert variant="warning">
+  <strong>Caution:</strong> You may never unsee this pattern across the web!
```

Comment:
It could help to make this alert text wrap to 2 lines. From all the examples, it's not clear that Figure and Body can be different heights.

Reply:
Updated the example